index.bs

```diff
@@ -249,26 +250,27 @@ stores and indexes. The following example shows one way to handle
 migrating from an older version of the database.
 
 ```js
-var request = indexedDB.open("library", 3); // Request version 3.
+const request = indexedDB.open("library", 3); // Request version 3.
+let db;
 
 request.onupgradeneeded = function(event) {
-  var db = request.result;
+  const db = request.result;
```

Comment:
Same as before: should const be here? I believe the goal was to set the variable in the outer scope context.

Reply:
ditto, should be in another PR